store/sqlstore/user_store.go

```diff
@@ -1361,3 +1361,470 @@ func (us SqlUserStore) InferSystemInstallDate() store.StoreChannel {
 		result.Data = createAt
 	})
 }
+
+// This function will run an sql query and return all relevant user interactions for the current user in the current mattermost team
+// Each result row will be of type UserInteraction (defined in model/user.go)
+// TO-DO: Break the query in this function into smaller, more maintainable queries perhaps using a query builder
```

Comment:
Good comment but change `TO-DO` to `TODO`. it's just how it is typically written. grep the code base for TODO you'll see a bunch of them.

Reply:
Changed.